docs/index.md

```diff
@@ -0,0 +1,16 @@
+% pytest_splunk_addon_ui_smartx documentation master file, created by
+% sphinx-quickstart on Fri Oct  9 15:31:49 2020.
+% You can adapt this file completely to your liking, but it should at least
+% contain the root `toctree` directive.
+
+# Welcome to pytest_splunk_addon_ui_smartx's documentation!
+
+```{toctree}
+:caption: 'Contents:'
+:maxdepth: 3
+
+overview
+how_to_use
+getting_started
+api_reference
+```
```

Comment:
It does not seem right. `overview.md` looks like a nice page to default to when you open a SmartX documentation.

Reply:
Updated the file.